### PR TITLE
Animating Morph Targets

### PR DESCRIPTION
**Pull request by emilian0**

### CODE REVIEW

File: specification/2.0/README.md [hunk 1]
```diff
@@ -1183,20 +1183,50 @@ The following examples show expected animations usage.
...
 
-Each of the animation's *samplers* defines the input/output pair: a set of floating point scalar values representing time; and a set of three-component floating-point vectors representing translation or scale, or four-component floating-point vectors representing rotation. All values are stored in a buffer and accessed via accessors. Interpolation between keys is performed using the interpolation formula specified in the `interpolation` property
+Each of the animation's *samplers* defines the input/output pair: a set of floating point scalar values representing time; and a set of three-component floating-point vectors representing translation or scale, or four-component floating-point vectors representing rotation, or scalar used to animate Morph Targets. All values are stored in a buffer and accessed via accessors. Interpolation between keys is performed using the interpolation formula specified in the `interpolation` property
```
Comment (lexaknyazev) [inline specification/2.0/README.md]: > or scalar used to animate Morph Targets

floating-point scalars used to animate Morph Targets

Comment (lexaknyazev) [inline specification/2.0/README.md]: > a set of floating point scalar values representing time

a set of floating point scalar values representing time in seconds (from #877)

Comment (lexaknyazev) [inline specification/2.0/README.md]: > interpolation formula

Maybe change to "interpolation method".

Comment (lexaknyazev) [inline specification/2.0/README.md]: Maybe add a note on using sparse arrays when it's beneficial.

**Comment by lexaknyazev** [comment]
Lgtm.
@pjcozzi, OK to merge?

**Comment by pjcozzi** [comment]
Thanks again, @emilian0!

**Pull request merged**